File: unidad1_conceptos_basicos/A1ejercicio.py
# n = 4
# fact = 1

# for i in range(1,n):
#     fact *= n
#     n -= 1
    
# print(fact)

def factorial(n):
    fact = 1
    if n == 0:
        return 1
    else:
        for i in range(1, n):
            fact *= n
            n -= 1
        return fact
    
print(factorial(6))

2+5

File: unidad1_conceptos_basicos/A1ejercicio.js
const factorial = (n) => {
    let fact = 1;
    if(n == 0)
        return 1;
    else{
        for (let i = 1; i<= n; i++)
            fact *= i;
        return fact;
    }
}

console.log(`El factorial es: ${ factorial(10) }`);

// El factorial es: 3628800

